ramanujan/cmf/known_cmfs/known_cmfs.py

```diff
@@ -1,5 +1,8 @@
 import sympy as sp
+import os
```

Comment:
keep it ordered - either all of the sympy imports before or all after the os imports